core/sync/sync.go

```diff
@@ -265,7 +265,7 @@ func (bs *bucketSynchronizer) addFileInfoToStore(addFileInfo domain.AddWatchFile
 	if err := bs.store.SetString(getOpenFileKey(addFileInfo.LocalPath), string(out)); err != nil {
 		return err
 	}
-	reverseKey := getOpenFileReverseKey(addFileInfo.BucketKey, addFileInfo.BucketPath)
+	reverseKey := getOpenFileReverseKey(addFileInfo.BucketSlug, addFileInfo.BucketPath)
```

Comment:
Does `getOpenFileReverseKey` need to filter by a db ID as well? This was done on a previous PR so might have missed that but since we can have the same bucket name in different threads (and both my files and shared with me files will have `personal` as the name and for broader API usage this might be needed, but maybe could be post-alpha).

Reply:
Added `dbID` to the cache in a40be09